docs/src/main/sphinx/release/release-450.md

```diff
@@ -0,0 +1,131 @@
+  Amazon EKS with Kubernetes by setting the
+  `trino.s3.use-web-identity-token-credentials-provider` configuration property. ({issue}`22162`)
+* Add support for exclusively using web identity for authentication when using
+  Amazon EKS with
+  [IAM roles](https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts.html)
+  by setting the
+  `s3.use-web-identity-token-credentials-provider` configuration property. ({issue}`22163`)
+
+## JDBC driver
+
+* Add support for the `assumeNullCatalogMeansCurrent` connection property. When
+  enabled, a `null` value for the `catalog` parameter in `DatabaseMetaData`
+  methods is assumed to mean the current catalog. If no current catalog is
+  set, the behaviour is unmodified. ({issue}`20866`)
+
+## BigQuery connector
+
+* Add support for name caching when the
+  `bigquery.case-insensitive-name-matching` configuration property is enabled. ({issue}`10740`)
+* {{breaking}} Automatically configure BigQuery scan parallelism, and remove the
+  `bigquery.parallelism` configuration property. ({issue}`22279`)
+
+## Cassandra connector
+
+* Fix incorrect behavior that uses one more partition than expected when
+  specifying a value for the `cassandra.partition-size-for-batch-select`
+  configuration property. ({issue}`21940`)
+
+## ClickHouse connector
+
+* Improve performance of `ORDER BY ... LIMIT` on non-textual types by pushing
+  execution down to the underlying database. ({issue}`22174`)
+
+## Delta Lake connector
+
+* Add support for concurrent `UPDATE`, `MERGE`, and `DELETE` queries. ({issue}`21727`)
+* Add support for using table statistics with `TIMESTAMP` types. ({issue}`21878`)
+* Add support for reading tables with
+  [type widening](https://docs.delta.io/latest/delta-type-widening.html). ({issue}`21756`)
+* Set the default value for the `s3.max-connections` configuration property
+  to 500. ({issue}`22209`)
+* Fix failure when reading a `TIMESTAMP` value after the year 9999. ({issue}`22184`)
+* Fix failure when reading tables with the unsupported `variant` type. ({issue}`22310`)
+
+## Hive connector
+
+* Add support for `integer` to `varchar` and `decimal` to `varchar` type 
+  coercion in unpartitioned tables. ({issue}`22246`, {issue}`22293`)
+* Add support for `double` to `varchar` type coercion in unpartitioned tables
+  using Parquet files. ({issue}`22277`)
+* Add support for `float` to `varchar` type coercion. ({issue}`22291`)
+* Set the default value for the `s3.max-connections` configuration property
+  to 500. ({issue}`22209`)
+
+## Hudi connector
+
+* Set the default value for the `s3.max-connections` configuration property
+  to 500. ({issue}`22209`)
+
+## Iceberg connector
+
+* Add support for reading Iceberg
+  [UniForm](https://docs.delta.io/latest/delta-uniform.html) tables. ({issue}`22106`)
+* Add support for the `TRUNCATE` statement. ({issue}`22340`)
+* Add support for V2 of the Nessie REST API. Previous behavior can be
```

Comment:
So its actually .. Change to use the V2 protocol by default..

Reply:
We didn't previously support V2, did we?

Reply:
no .. but we are not only adding support .. we are switching as well..

Reply:
so maybe that is also a breaking change?